modules/javafx.web/src/main/java/com/sun/javafx/webkit/prism/WCPathImpl.java

```diff
@@ -344,4 +345,48 @@ public void transform(double mxx, double myx,
         }
         path.transform(BaseTransform.getInstance(mxx, myx, mxy, myy, mxt, myt));
     }
+
+    @Override
+    public boolean strokeContains(double x, double y,
+                                  double thickness, double miterLimit,
+                                  int cap, int join, double dashOffset,
+                                  double[] dashArray) {
+
+        BasicStroke stroke = new BasicStroke(
+            (float) thickness, cap, join, (float) miterLimit);
+
+        boolean hasDash = false;
+        for (double d : dashArray) {
+            if (d > 0) {
+                hasDash = true;
+                break;
+            }
+        }
```

Comment:
This "contains non-zero element" check is already done in `BasicStroke#set(double[], float)`. I think you can change the constructor to `BasicStroke(thickness, cap, join, meterLimit, dashArray, dashOffest)`.